packages/lodestar-utils/src/buf64copy.ts

```diff
@@ -0,0 +1,178 @@
+export function copyFromBuf64LE(sbuffer: Buffer, tbuffer: Buffer, length: number): void {
```

Comment:
might be better named `source` and `target`. We know they're a buffer from their type :slightly_smiling_face: